docs/reference/data-analysis/aggregations/search-aggregations-bucket-adjacency-matrix-aggregation.md

```diff
@@ -88,6 +88,12 @@ The response contains buckets with document counts for each filter and combinati
 }
 ```
 
+%  TESTRESPONSE[s/"took": 9/"took": $body.took/]
+
+%  TESTRESPONSE[s/"_shards": \.\.\./"_shards": $body._shards/]
+
+%  TESTRESPONSE[s/"hits": \.\.\./"hits": $body.hits/]
```

Comment:
Not sure if it was intentional to have empty lines between these, but previously they were stacked, which seems a little tidier 🤷